middleware/CORSMiddleware.go

```diff
@@ -0,0 +1,18 @@
+package middleware
+
+import (
+	"net/http"
+)
+
+func CORSAccessControlHeadersMiddleware(next http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Access-Control-Allow-Origin", "*")
```

Comment:
I think we actually want to use an allowlist here. `*` will work for development but for production we want a specific value. What I've done in the past is actually implement a map of allow origins, see if the incoming origin is in that map, and, if so, return just that origin here.